specification/metrics/effectivecost.md

```diff
@@ -1,13 +1,13 @@
 # Effective Cost
 
-Effective cost represents the sum of the amortized upfront fees, amortized recurring fees, and the usage cost, including discounts,  of the line item. This column is often used to show spending trends that include on demand rates, effective commitment-based discount rates, recurring commitments, as well as amortization of upfront fees. The fee and upfront amortization portion of this column's value should be proportional to the number of units for the line item and the time granularity of the data. The currency for the value specified in Effective Cost can be found in the [Billing Currency](#billingcurrency) column. The aggregated Effective Cost for a billing period may not match the charge received on the invoice for the same billing period.
+The Effective Cost represents a charge inclusive of the impacts of all reduced rates and discounts, augmented with the amortization of upfront charges (one-time or recurring). This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends. The amortization of upfront charges portion of this metric's value should be proportional to the [Quantity In Pricing Unit](#quantityinpricingunit) and the time granularity of the data.
```

Comment:
Suggestions only. Use all or part of the updates as you see fit.

```suggestion
The Effective Cost represents a charge inclusive of the impacts of all reduced rates and discounts, augmented with the amortization of relevant pre-paid fees (one-time or recurring) paid to cover future eligible charges. The amortized  portion included should be proportional to the [Quantity In Pricing Unit](#quantityinpricingunit) and the time granularity of the data. This cost is denominated in the [Billing Currency](#billingcurrency). The Effective Cost is commonly utilized to track and analyze spending trends.
```

Reply:
We've used the term `amortization of upfront charges (one-time or recurring)` in remaining cost and unit price definitions. Should we rephrase all occurrences to `amortization of relevant pre-paid fees (one-time or recurring)`? 
Do we need an additional PR to address this in the BilledCost spec? :)

Reply:
Technically, not all amortized purchases will be "upfront" or "pre-paid". It might be more inclusive to say `amortization of relevant purchases`.